### PR TITLE
🏗 Fix bot account name for owners check

### PR DESCRIPTION
**Pull request by rsimha**
Context: https://github.com/ampproject/amphtml/pull/34676#issuecomment-854039159

This now passes the schema check.

### CODE REVIEW

File: OWNERS
Comment (rcebulko) [inline OWNERS:43]: Since we now support bots as user owners, let's make this the same as the one in the approvers group. The effect is the same (an approving PR review from a) an owner and b) a reviewer-group member) but it's a bit simpler to look at
```suggestion
      owners: [{name: 'renovate-approve[bot]'}],
```

Comment (rsimha) [reply OWNERS:43]: Some additional context:
1. All upgrades are authored by `renovate[bot]`.
2. Some upgrades are auto-approved by `renovate-approve[bot]`.

If 1 is true but not 2, we still want a build on-duty who is a reviewer but not an owner (e.g. a subpackage upgrade) to be able to manually approve and merge a PR after CI is green.

Based on this, I'd imagine it makes sense to leave the code as is?

**Review by rcebulko — APPROVED** [review]

**Review by jridgewell — APPROVED** [review]

**Pull request merged**
